database/migrations/2017_04_20_100848_create_monsters_table.php

```diff
@@ -12,10 +12,20 @@ class CreateMonstersTable extends Migration
      */
     public function up()
     {
-        Schema::create('monsters', function (Blueprint $table) {
+        $driver = DB::getDriverName();
+
+        switch ($driver) {
+            case 'pgsql': $columnType = 'BYTEA';
+            break;
+            case 'mysql': $columnType = 'MEDIUMBLOB';
+        }
```

Comment:
PHP 8 `match` would fit so well here 😅

```php8
$columnType = match($driver) {
    'pgsql' => 'BYTEA',
    'mysql' => 'MEDIUMBLOB',
};
```

I know we can't use it, just to let you guys know 😅
See you in a year or two `match` 👋

Reply:
Way cleaner. I am particularly excited about the null safe operator.